### PR TITLE
FISH-322 Update Email Notifier to New API

### PR DESCRIPTION
**Pull request by MattGill98**
- Renamed the email-notifier-configure command to the more common set-
syntax.
- Renamed the ambiguous 'to' option to 'recipient'.
- Fixed remaining compilation issues

The following PRs will be to remove this notifier and put it in the new suite. This PR is here for simplicity so that the code can be reviewed in one place before it's moved to the new repo.

### CODE REVIEW

**Comment by MattGill98** [comment]
jenkins test please

File: appserver/payara-appserver-modules/notification-email-core/src/main/java/fish/payara/extras/notifiers/email/EmailNotifier.java [hunk 1]
```diff
@@ -0,0 +1,128 @@
...
+        try {
+            Message mime = new MimeMessage(session);
+            mime.setRecipients(Message.RecipientType.TO, InternetAddress.parse(configuration.getRecipient()));
+            mime.setSubject(subject);
+            mime.setText(message);
+
+            Transport.send(mime);
+            LOGGER.log(Level.FINE, "Email successfully sent");
+        } catch (IllegalArgumentException e) {
+            LOGGER.log(Level.SEVERE, "mail configuration properties are not provided carefully.", e);
```
Comment (AlanRoth) [inline appserver/payara-appserver-modules/notification-email-core/src/main/java/fish/payara/extras/notifiers/email/EmailNotifier.java:100]:
```suggestion
            LOGGER.log(Level.SEVERE, "Email configuration properties were not provided correctly.", e);
```

Comment (AlanRoth) [reply appserver/payara-appserver-modules/notification-email-core/src/main/java/fish/payara/extras/notifiers/email/EmailNotifier.java:100]: They could have taken all the care in the world to put in the wrong properties

**Comment by MattGill98** [comment]
jenkins test please

**Review by Cousjava — APPROVED** [review]

**Pull request merged**